.github/workflows/cargo.yml

```diff
@@ -40,9 +40,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: mozilla-actions/sccache-action@v0.0.5
+      - uses: mozilla-actions/sccache-action@v0.0.9
         with:
-          version: "v0.8.1"
```

Comment:
we hit a rebase conflict because v2.1 was still on version v0.8.1 wheras master/v2.2 were on v0.9.1. I'm assuming we're okay jumping up to v0.10.0, right?

Reply:
i don't believe we have a choice